Dockerfile

```diff
@@ -173,25 +185,29 @@ RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
 # Install Rust 1.51.0
 ENV RUSTUP_HOME=/opt/rust \
   PATH="${PATH}:/opt/rust/bin"
+USER dependabot
+RUN mkdir -p "$RUSTUP_HOME"
```

Comment:
Noice: since `/opt` is already owned by `dependabot`, doing the `USER` instruction first means we can skip the `chown` part of the `mkdir && chown` from other ecosystems.